### PR TITLE
Fetch data

### PR DESCRIPTION
**Pull request by Shaheryar0054**
- Used the following external API: [API Ninjas](https://api-ninjas.com/api/quotes)
- Created a new component for displaying a quote
- Fetch data inside of a new component
- Show a loading state
- Show an error state

### CODE REVIEW

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import './Calculator.css';
+import './calculator.css';
 import calculate from '../logic/calculate';
```
Comment (thecodechaser) [inline src/Components/Calculator.js]: - [ ] Great job on the project requirements but there's one issue when I'm trying to start the app, getting the below error. Please make sure to fix it.
   - To fix this issue, Please replace the line `import './calculator.css';` with `import './Calculator.css';`

image

File: src/Components/Quote.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import './calculator.css';
+
```
Comment (thecodechaser) [inline src/Components/Quote.js]: - [ ] The same issue also happening here when I'm trying to start the app, getting the below error. Please make sure to fix it.
   - To fix this issue, Please replace the line `import './calculator.css';` with `import './Calculator.css';`

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Shaheryar0054👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Shaheryar0054 , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**